### ISSUE TITLE
Expected Behaviour with verticalPadding?

### ISSUE THREAD

**Issue by btanner** [scalessec/Toast-Swift#3]
I have not used Toast on Android or elsewhere. I came across your project and it seemed like an easy way to pop up status message in my app.

I have a dark section across the bottom of my App, so I want to use .Bottom but I thought I could use verticalPadding to raise the location up higher on the screen.

I am not using images or titles, just messages.

This line is causing a problem for my use case:

``Swift
       let wrapperHeight = max((messageRect.origin.y + messageRect.size.height + style.verticalPadding), (imageRect.size.height + (style.verticalPadding * 2.0)))
``

Basically, if I set the padding to something like 120 to push the Toast box up, it also makes the wrapper view large, so a label with height 20 is shown on a gray background that has a large height.

Seems like a bug?

![toast-screencap](https://cloud.githubusercontent.com/assets/9030869/13375442/102a6c52-dd65-11e5-8227-a45691024962.png)

**Comment by btanner** [comment]
Quick solution is just to add a new property called verticalOffset which is used like
``Swift
    private func centerPointForPosition(position: ToastPosition, toast: UIView) -> CGPoint {
        let padding: CGFloat = ToastManager.shared.style.verticalOffset
``

**Comment by scalessec** [comment]
The documentation states:
````
/**
     The spacing from the vertical edge of the toast view to the content. When a title
     is present, this is also used as the padding between the title and the message.
     Default is 10.0.
*/
var verticalPadding: CGFloat = 10.0
````
Vertical padding is the space between from the top of the toast view to the text (and also from the bottom of the text to the bottom of the toast view). Looking at your screenshot, this is the expected behavior. 

**When you want is a custom presentation position.**

Instead of using `.Bottom` you can just pass a `CGPoint` using this method instead:
````
// notice this method takes a CGPoint, and not a ToastPosition enum
func makeToast(message: String, duration: NSTimeInterval, position: CGPoint)
````
In this case, the position is the center point for the toast view. If the height of the toast view will vary (eg: if you potentially have multiple lines of text), you could first create the toast view with `toastViewForMessage(message:title:image:style:)` so that you can determine the height of the view, and then you can use `showToast(toast:duration:position:completion:)` (using the CGPoint version) to present the view at the correct center position.
